app/views/moments/show.html.erb

```diff
@@ -1,5 +1,14 @@
 <% title @moment.name %>
 <%= render :partial => '/moments/info', locals: { data: @moment } %>
+<% if Rails.configuration.secret_share_enabled && @moment.owned_by?(current_user) %>
```

Comment:
We should specify here that the secret share URL expires in a day.

Reply:
This feature has a default of one day, it will be changed in the next feature, so you can choose time/date of expiry. I can add a comment to the code for now if you like.

Reply:
@SophieRoseMcDonald Ah that's fine then, no need for comment! Thanks!